messages.proto

```diff
@@ -256,12 +256,21 @@ message UnsupportedMessageResponse {
 
 /// Request for caching a file
 message CacheFileRequest {
+    enum FileStatus {
+        CHANGED = 0;
```

Comment:
Some comments on the enum values  will help us autogenerate the [api docs](https://github.com/getgauge/gauge-proto/blob/master/doc/gauge-proto-doc.md).